### ISSUE TITLE
White gap at screen bottom after Update to NC 14.0.0 Beta 3

### ISSUE THREAD

**Issue by MarcMichalsky** [nextcloud/files_pdfviewer#92]
### Steps to reproduce
1. Update Nextcloud to 14.0.0 Beta 3
2. Open any PDF file with the pdfviewer

### Expected behaviour
Display the PDF as usual

### Actual behaviour
There is a large white gap between the displayed document and the bottom of the screen.

### Server configuration

**Operating system**:
16.04.4 LTS

**Web server:**
Apache

**Database:**
MariaDB

**PHP version:**
7.0.30

**Nextcloud version:**
14.0.0 Beta 3

**Where did you install Nextcloud from:**
https://nextcloud.com/install/#instructions-server

**List of activated apps:**
```
Enabled:
  - accessibility: 1.0.1
  - bruteforcesettings: 1.1.0
  - calendar: 1.6.1
  - cloud_federation_api: 0.0.1
  - comments: 1.4.0
  - dav: 1.6.0
  - encryption: 2.1.0
  - external: 3.1.0
  - federatedfilesharing: 1.4.0
  - federation: 1.4.0
  - files: 1.9.0
  - files_external: 1.5.0
  - files_markdown: 2.0.4
  - files_pdfviewer: 1.3.2
  - files_rightclick: 0.8.4
  - files_sharing: 1.6.2
  - files_texteditor: 2.6.0
  - files_trashbin: 1.4.1
  - files_versions: 1.7.1
  - files_videoplayer: 1.3.0
  - firstrunwizard: 2.3.0
  - gallery: 18.1.0
  - groupfolders: 1.3.3
  - logreader: 2.0.0
  - lookup_server_connector: 1.2.0
  - nextcloud_announcements: 1.3.0
  - notifications: 2.2.1
  - oauth2: 1.2.1
  - onlyoffice: 1.3.0
  - password_policy: 1.4.0
  - provisioning_api: 1.4.0
  - registration: 0.4.3
  - serverinfo: 1.4.0
  - sharebymail: 1.4.0
  - support: 1.0.0
  - survey_client: 1.2.0
  - systemtags: 1.4.0
  - theming: 1.5.0
  - twofactor_backupcodes: 1.3.0
  - twofactor_u2f: 1.6.1
  - updatenotification: 1.4.1
  - user_external: 0.4
  - workflowengine: 1.4.0
Disabled:
  - activity
  - admin_audit
  - admin_notifications
  - audioplayer
  - files_downloadactivity
  - files_linkeditor
  - files_trackdownloads
  - ransomware_protection
  - spreed
  - twofactor_totp
  - user_ldap
```

```
**Nextcloud configuration:**

<?php
$CONFIG = array (
  'instanceid' => '***',
  'passwordsalt' => '***',
  'secret' => '***',
  'trusted_domains' => 
  array (
    0 => '***',
  ),
  'datadirectory' => '/var/www/***-data',
  'overwrite.cli.url' => 'http://***',
  'dbtype' => 'mysql',
  'version' => '14.0.0.15',
  'dbname' => '***',
  'dbhost' => 'localhost',
  'dbport' => '',
  'dbtableprefix' => 'oc_',
 'dbuser' => '***',
  'dbpassword' => '***',
  'installed' => true,
  'htaccess.RewriteBase' => '/﻿',
  'memcache.distributed' => '\\OC\\Memcache\\Redis',
  'memcache.local' => '\\OC\\Memcache\\Redis',
  'memcache.locking' => '\\OC\\Memcache\\Redis',
  'redis' => 
  array (
 'host' => 'localhost',
    'port' => 6379,
  ),
  'mail_from_address' => 'nextcloud',
  'mail_smtpmode' => 'smtp',
  'mail_smtpauthtype' => 'LOGIN',
  'mail_domain' => '***',
  'mail_smtpsecure' => 'ssl',
  'mail_smtphost' => 'smtp.strato.de',
'mail_smtpport' => '465',
  'mail_smtpauth' => 1,
  'mail_smtpname' => 'nextcloud@***',
  'mail_smtppassword' => '***',
  'maintenance' => false,
  'theme' => '',
  'loglevel' => 2,
  'default_language' => 'de',
  'updater.release.channel' => 'beta',
  'updater.secret' => '***',
);

```

### Client configuration
**Browser:**
Firefox 61.0.1 (64-Bit)

**Operating system:**
Ubuntu 18.04

#### Nextcloud log (data/owncloud.log)
```

```

#### Browser log
```
Content Security Policy: Die Einstellungen der Seite haben das Laden einer Ressource auf self blockiert ("script-src"). Source: (function (NAVIGATOR, OBJECT) {

    OBJ.... files_pdfviewer:1
Content Security Policy: Die Einstellungen der Seite haben das Laden einer Ressource auf self blockiert ("script-src"). Source: (function (DOCUMENT, dispatchEvent, CUST.... files_pdfviewer:1
Content Security Policy: Die Einstellungen der Seite haben das Laden einer Ressource auf self blockiert ("script-src"). Source: call to eval() or related function blocked by CSP.
```
#### Screenshot
![Screenshot](https://marc-michalsky.de/share/screenshot_bug_pdfviewer.png)

**Comment by quentinDupont** [comment]
I have the same problem.

**Comment by MorrisJobke** [comment]
@danxuliu You looked at this lately. Did you also experienced this?

**Comment by mathiasconradt** [comment]
@MorrisJobke I got the confirmation from another user that it happens on IE11 (and only there)

**Comment by danxuliu** [comment]
This bug should have been fixed in NC 14.0.0 Beta 4; see nextcloud/server#10641

>  I got the confirmation from another user that it happens on IE11 (and only there)

That is strange, because in IE11 they should be experiencing other bugs :-P If the PDF is opened from the Files app, they should be experiencing the bug fixed by #95; if the PDF is opened in the public share page then the viewer should appear very narrow (due to what looks like a typo in the rules for IE 11 in the server [that forces `app-content` to have the width of the navigation bar](https://github.com/nextcloud/server/blob/0274507cb10b53cfd38bd0ae3810c26cdb172ad1/core/css/ie.scss#L5-L7) and the rules for IE 11 in the PDF viewer [that displays `content` as a block](https://github.com/nextcloud/files_pdfviewer/blob/fc4a395a504f3e09e687808c6ffa0e784f0fc4e0/css/style.css#L6) overriding [the flex layout for `content`](https://github.com/nextcloud/server/blob/8f9571bc3b6f2ced6383e2a617583399f7efbbdd/core/css/apps.scss#L589) from the server (and thus causing the typo to manifest as a bug in the PDF viewer but not in other apps)).

**Comment by MorrisJobke** [comment]
#95 got back ported to 14.0.2 in #98 -> closing then.
